docusaurus/docs/iOS/controllers/channels.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Channel Controllers
+---
+
+## ChannelListController
+
+`ChatChannelListController` allows you to observe a list of chat channels based on the provided query and to paginate channels.
+
+### ChannelList Delegate
+
+Classes that conform to the `ChatChannelListControllerDelegate` protocol will receive changes to the queried list of channels.
+
+```swift
+func controller(
+    _ controller: ChatChannelListController,
+    didChangeChannels changes: [ListChange<ChatChannel>]
+)
+```
+
+## ChannelController
+
+`ChatChannelController` allows you to observe and mutate data for one channel.
+
+### Channel Delegate
```

Comment:
Is the `ChatChannelControllerDelegate` protocol something we have documented in the API reference for iOS? If yes, we should link to it from this section.

Reply:
see below (protocol are similar interfaces)